recommenders/models/rbm/rbm.py

```diff
@@ -554,7 +554,7 @@ def train_test_precision(self, xtst):
 
             self.sess.run(
                 self.iter.initializer,
-                feed_dict={self.vu: xtst, self.batch_size: xtst.shape[0]},
+                feed_dict={self.vu: xtst, self.batch_size: self.minibatch},
```

Comment:
Changing batch size here resolves OOM error during model fitting for 10M/ 20M movielens datasets.